Gemfile.lock

```diff
@@ -67,11 +85,19 @@ GEM
       ffi (~> 1.9)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
+    thread_safe (0.3.6)
+    tzinfo (1.2.7)
+      thread_safe (~> 0.1)
     unicode-display_width (1.7.0)
+    zeitwerk (2.4.1)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   jekyll
   jekyll-theme-opensuse!
+  jemoji (~> 0.12)
+
+BUNDLED WITH
+   2.1.4
```

Comment:
d10c9b53c922dcca4c5fa89e86e77ed0564b8265 explicitely removed the `BUNDLED WITH` (unfortunately without any details in the commit message).

IIRC we had some fun[tm] with it in the past because of different bundler versions in Leap and Tumbleweed (leading to "wrong version" failures), and in the end removed the `BUNDLED WITH` to avoid the versioned dependency. However, I'm still not familiar with ruby, so please take this with a grain of salt ;-)

Reply:
I honestly don't understand why `BUNDLED WITH` was removed. It specifies the version of Bundler used to create the `Gemfile.lock` file. When installing the dependencies Bundler fails if your version is older than the version that created the file. This way you ensure everything works. Removing the version is IMO a bad idea, as when I install the project I don't know anymore which version I should use (and the same for production). You are removing the specification of the version dependency, but the dependency still exists and the something could break. I think we should keep` BUNDLE WITH` (in general we shouldn't modify this file manually) and either:
- Update bundle in production to use a version higher or equal to the one used by the developer (or even better the newest version of bundler)
- Document the desired version of bundle that the developers should use

Updating bundle is not a big deal and I think it is a good idea to use a newer version, so I would go for option one, except if there is a reason not to do it this way (I don't know how the deployment is done).